README.md

```diff
@@ -165,11 +171,21 @@ Below is an example of how you should add the package.
 ```json
 ...
 "devDependencies": {
-  "@ory/elements": "*"
+"@ory/elements": "*"
 }
 ...
 ```
 
+### Default Translations
+
+Ory Elements comes with default translations for any language that was added
+yet. They are located in the `src/locales` directory. The English messages are
+extracted from the code and merged with Kratos messages. To update them, run
```

Comment:
```suggestion
Ory Elements comes with default translations for a few language, such as English, German and Spanish.
They are located in the `src/locales` directory. The English messages are
extracted from the code and merged with Kratos messages. To update them, run
```

Reply:
not sure what `code` the messages are extracted from. Is it from the Ory Elements codebase?